makeflow/src/makeflow.c

```diff
@@ -651,7 +651,11 @@ static void makeflow_node_complete(struct dag *d, struct dag_node *n, struct bat
 
 	struct list *outputs = makeflow_generate_output_files(n, wrapper, monitor);
 
-	if(info->exited_normally && info->exit_code == 0) {
+
+	if(getenv("CCTOOLS_LOOP_DEV_FULL")) {
+		job_failed = 1;
```

Comment:
I can see why you did it this way, but let's not get into the habit of passing information back via an environment variable.
Add an element to `batch_job_info` instead, which is where we send back detailed info about jobs.